plugin/kubernetes/setup.go

```diff
@@ -201,6 +201,12 @@ func kubernetesParse(c *caddy.Controller) (*Kubernetes, dnsControlOpts, error) {
 					return nil, opts, c.Errf("ttl must be in range [5, 3600]: %d", t)
 				}
 				k8s.ttl = uint32(t)
+			case "transfer":
+				dests, err := TransferParse(c)
```

Comment:
Can you not just call `file.TransferParse(c, false)`

Reply:
We could do that, but I didn't want to create a dependency between the file and Kubernetes plugin.

Reply:
Also, the `file.TransferParse` supports transfer from, which we don't want to support in the Kubernetes plugin.

Reply:
But I guess we could just ignore/err if we get froms returned. So that's an option.

Reply:
secondary also depends on file, if it becomes cyclic we move to pkg/...
transfer from is an error when called `secondary = false`